src/workspaces/common/state/useCloudEnvironmentPolling.ts

```diff
@@ -43,7 +44,6 @@ export const useCloudEnvironmentPolling = (
   };
   const load = async (maybeStale?: boolean): Promise<void> => {
     try {
-      setIsLoadingCloudEnvironments(true);
```

Comment:
Can you provide some context on why this is removed?

Reply:
Oh, yeah! I took this out so that only the initial fetch would block user actions. Subsequent polls might occur every 10 seconds if a runtime is in a transition state, and I didn't want to disable UI interaction during those times. Actions performed on slightly stale data are not nearly as dangerous as those on uninitialized state.